charts/pulsar/values.yaml

```diff
@@ -551,7 +552,9 @@ zookeeper:
   annotations:
     prometheus.io/scrape: "true"
     prometheus.io/port: "8000"
-  securityContext: {}
+  securityContext: 
+    fsGroup: 0
```

Comment:
why is fsGroup 0? (does it have to be 0 or is it randomly selected?)

Reply:
Not very sure about this part, just keep consistent with https://github.com/apache/pulsar-helm-chart/blob/master/charts/pulsar/values.yaml#L349.
But the weird thing is even if not `securityContext` set on zk and bk, in my testing they are still okay to run.